src/cli/generator/snapshots/tailcall__cli__generator__generator__test__generator_spec__src__cli__generator__tests__fixtures__generator__json_proto_mix_config.json.snap

```diff
@@ -47,25 +47,25 @@ type Geo {
   lng: String
 }
 
-type News @tag(id: "news.News") {
+type NewsNewsServiceGetMultipleNew @tag(id: "news.NewsList") {
+  news: [NewsNewsServiceGetNew]
+}
+
+type NewsNewsServiceGetNew @tag(id: "news.News") {
   body: String
   id: Int
   postImage: String
   status: news__Status
   title: String
 }
 
-type NewsNewsServiceGetMultipleNew @tag(id: "news.NewsList") {
-  news: [News]
-}
-
 type Query {
-  news__NewsService__AddNews(news: news__NewsInput!): News! @grpc(body: "{{.args.news}}", method: "news.NewsService.AddNews")
+  news__NewsService__AddNews(news: news__NewsInput!): NewsNewsServiceGetNew! @grpc(body: "{{.args.news}}", method: "news.NewsService.AddNews")
   news__NewsService__DeleteNews(newsId: news__NewsId!): Empty! @grpc(body: "{{.args.newsId}}", method: "news.NewsService.DeleteNews")
-  news__NewsService__EditNews(news: news__NewsInput!): News! @grpc(body: "{{.args.news}}", method: "news.NewsService.EditNews")
+  news__NewsService__EditNews(news: news__NewsInput!): NewsNewsServiceGetNew! @grpc(body: "{{.args.news}}", method: "news.NewsService.EditNews")
   news__NewsService__GetAllNews: NewsNewsServiceGetMultipleNew! @grpc(method: "news.NewsService.GetAllNews")
   news__NewsService__GetMultipleNews(multipleNewsId: news__MultipleNewsId!): NewsNewsServiceGetMultipleNew! @grpc(body: "{{.args.multipleNewsId}}", method: "news.NewsService.GetMultipleNews")
-  news__NewsService__GetNews(newsId: news__NewsId!): News! @grpc(body: "{{.args.newsId}}", method: "news.NewsService.GetNews")
+  news__NewsService__GetNews(newsId: news__NewsId!): NewsNewsServiceGetNew! @grpc(body: "{{.args.newsId}}", method: "news.NewsService.GetNews")
```

Comment:
This is a significant degradation in the quality of generated configuration.

Reply:
Make this feature configurable. Disable it in GRPC and enable it in REST by default.